_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,639 @@
+| [sourcecolumn](#sourcecolumn)             | any                        | The source column to provide data of the described column                                                                                    | *id*         | `-`[\*\*](#metadata-star-star)            |
+| [default](#default)                       | any atomic type            | Default value to use in case data are missing                                                                                                | *0*          | `-`[\*\*](#metadata-star-star)            |
+| [pattern](#pattern)                       | date & timestamp           | Pattern for the date or timestamp representation                                                                                             | *dd.MM.yy*   | *yyyy-MM-dd HH:mm:ss* **/** *yyyy-MM-dd* |
+| [timezone](#timezone)                     | timestamp (also date)      | The time zone of the timestamp when that is not part of the pattern (NB! for date it can return unexpected results)                          | *US/Pacific* | *UTC*[\*\*\*](#metadata-star-star-star)    |
+| [pattern](#pattern)                       | any numeric type           | Pattern for the number representation                                                                                                        | \#,\#\#0.\#  | `-`[\*\*](#metadata-star-star)            |
+| [decimal_separator](#decimal_separator)   | any numeric type           | The character separating the integer and the fractional parts of the number                                                                  | *,*          | *.*                                      |
+| [grouping_separator](#grouping_separator) | any numeric type           | Character to mark boundaries between orders of magnitude, usually to mark thousands, millions etc.                                           | *\_*         | *,*                                      |
+| [minus_sign](#minus_sign)                 | any numeric type           | Character to mark the number is negative.                                                                                                    | *N*          | *-*                                      |
+| [allow_infinity](#allow_infinity)         | float & double             | Flag indicating if the column accepts infinity as a value (also too big/far negative numbers are converted to become *infinity*/*-infinity*) | *true*       | *false*                                  |
+| [radix](#radix)                           | long, integer, short, byte | The base of the numbers provided                                                                                                             | *hex*        | *10*                                     |
+
+**NB!** All values in _metadata_ have to be entered as *string*. Even in cases
+they would conform to other types, like number or Boolean.
+
+<a name="#metadata-star" />\* Value used if nothing is specified in _metadata_
```

Comment:
For some reason, these don't work for me.

Reply:
Found the reason - the anchor name had a # character at the beginning.